### PR TITLE
Update itemGroups code examples, tests and types

### PR DESCRIPTION
**Pull request by mocca102**

### CODE REVIEW

File: src/types/catalog.d.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export interface AddItemGroupParameters {
 }
 
 export interface AddItemGroupsParameters {
-  itemGroups: ItemGroup[];
+  itemGroups: Omit<ItemGroup, 'children'>[];
```
Comment (stanlp1) [inline src/types/catalog.d.ts:78]: Why do we omit children here?

**Review by stanlp1 — APPROVED** [review]
lgtm minor comment about types

File: src/types/catalog.d.ts (hunk 1 above)
Comment (mocca102) [reply src/types/catalog.d.ts:78]: It's only part of the response not the request

Comment (mocca102) [reply src/types/catalog.d.ts]: You're correct it shouldn't be omitted I think I confused it with addItemGroup singular. Fixed!

**Review by esezen — APPROVED** [review]
LGTM!

**Pull request merged**